src/model_io/codecs/src/ModelExporter.cpp

```diff
@@ -21,7 +21,8 @@ namespace iDynTree
 ModelExporterOptions::ModelExporterOptions()
 : baseLink(""),
     exportFirstBaseLinkAdditionalFrameAsFakeURDFBase(true),
-    robotExportedName("iDynTreeURDFModelExportModelName") {}
+    robotExportedName("iDynTreeURDFModelExportModelName"),
+    xmlBlobs{""} {}
```

Comment:
Why we put an empty string in the vector?

Reply:
You are right it doesn't have sense, I changed it initializing w/ an empty vector